test/init.js

```diff
@@ -4,35 +4,40 @@
 // License text available at https://opensource.org/licenses/MIT
 
 'use strict';
-// TODO: used for testing support for parallel testing on ci.strongloop.com which
-// provides MYSQL_* env vars instead of TEST_MYSQL_* env vars.
-process.env.TEST_MYSQL_USER = process.env.TEST_MYSQL_USER || process.env.MYSQL_USER;
-process.env.TEST_MYSQL_PASSWORD = process.env.TEST_MYSQL_PASSWORD || process.env.MYSQL_PASSWORD;
-process.env.TEST_MYSQL_HOST = process.env.TEST_MYSQL_HOST || process.env.MYSQL_HOST;
-process.env.TEST_MYSQL_PORT = process.env.TEST_MYSQL_PORT || process.env.MYSQL_PORT;
-
-module.exports = require('should');
 
 var DataSource = require('loopback-datasource-juggler').DataSource;
 
 var config = require('rc')('loopback', {test: {mysql: {}}}).test.mysql;
-console.log(config);
-global.getConfig = function(options) {
-  var dbConf = {
-    host: process.env.TEST_MYSQL_HOST || config.host || 'localhost',
-    port: process.env.TEST_MYSQL_PORT || config.port || 3306,
+
+if (process.env.CI) {
+  process.env.TEST_MYSQL_HOST =
+    process.env.TEST_MYSQL_HOST || process.env.MYSQL_HOST || 'localhost'
+  process.env.TEST_MYSQL_PORT =
+    process.env.TEST_MYSQL_PORT || process.env.MYSQL_PORT || 3306;
+  process.env.TEST_MYSQL_USER =
+    process.env.TEST_MYSQL_USER || process.env.MYSQL_USER || 'test';
+  process.env.TEST_MYSQL_PASSWORD =
+    process.env.TEST_MYSQL_PASSWORD || process.env.MYSQL_PASSWORD || 'test';
+  config = {
+    host: process.env.TEST_MYSQL_HOST,
+    port: process.env.TEST_MYSQL_PORT,
+    username: process.env.TEST_MYSQL_USER,
+    pasword: process.env.TEST_MYSQL_PASSWORD,
     database: 'myapp_test',
-    username: process.env.TEST_MYSQL_USER || config.username,
-    password: process.env.TEST_MYSQL_PASSWORD || config.password,
-    createDatabase: true,
+    createDatabase: true
   };
+}
 
+module.exports = require('should');
+
+global.getConfig = function(options) {
+  var settings = config;
```

Comment:
Creating an alias for `config` has no benefit here, it will just make some people not notice that you are actually modifying the shared `config` when `settings` is manipulated.